app/src/main/res/values/strings.xml

```diff
@@ -176,6 +177,8 @@
     <string name="auto_refresh_covers">Automatically refresh covers</string>
     <string name="auto_refresh_covers_summary">Refresh covers in library as well
         when updating library</string>
+    <string name="show_hide_scrollbar">Show scrollbar</string>
+    <string name="edit_text_show_hide_scrollbar">Show library fast scrollbar :</string>
```

Comment:
i don't believe this is in correct location.  It should be with the other display items.  under <!-- Library Display -->

Reply:
also remove the old string if its not being used anymore